compiler/rustc_codegen_ssa/Cargo.toml

```diff
@@ -8,9 +8,9 @@ edition = "2024"
 ar_archive_writer = "0.4.2"
 bitflags = "2.4.1"
 bstr = "1.11.3"
-# Pinned so `cargo update` bumps don't cause breakage. Please also update the
-# `cc` in `rustc_llvm` if you update the `cc` here.
-cc = "=1.2.16"
+# `cc` updates often break things, so we pin it here. Cargo enforces "max 1 semver-compat version
+# per crate", so if you change this, you need to also change it in `rustc_llvm`.
+cc = "=1.2.30"
```

Comment:
FWIW, we could just remove the pin here and rely on `rustc_llvm` pinning the version for the entire workspace... :shrug: